docs/using-gitbase/schema.md

```diff
@@ -0,0 +1,251 @@
+
+### refs
+``` sql
++---------------+------+
+| name          | type |
++---------------+------+
+| repository_id | TEXT |
+| ref_name      | TEXT |
+| commit_hash   | TEXT |
++---------------+------+
+```
+This table contains all hash [git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References) and the symbolic reference `HEAD` from all the repositories.
+
+### commits
+``` sql
++---------------------+-----------+
+| name                | type      |
++---------------------+-----------+
+| repository_id       | TEXT      |
+| commit_hash         | TEXT      |
+| commit_author_name  | TEXT      |
+| commit_author_email | TEXT      |
+| commit_author_when  | TIMESTAMP |
+| committer_name      | TEXT      |
+| committer_email     | TEXT      |
+| committer_when      | TIMESTAMP |
+| commit_message      | TEXT      |
+| tree_hash           | TEXT      |
+| commit_parents      | JSON      |
++---------------------+-----------+
+```
+
+Commits contains all the [commits](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects#_git_commit_objects) from all the references from all the repositories, not duplicated by repository. Note that you can have the same commit in several repositories. In that case, the commit will appears two times on the table, one per repository.
+
+> Note that this table is not only showing `HEAD` commits but all the commits on the repository (that can be a lot more than the commits on `HEAD` reference).
+
+### blobs
+```sql
++---------------+-------+
+| name          | type  |
++---------------+-------+
+| repository_id | TEXT  |
+| blob_hash     | TEXT  |
+| blob_size     | INT64 |
+| blob_content  | BLOB  |
++---------------+-------+
+```
+
+This table exposes blob objects, that are the content without path from files.
+
+> Note that this table will return all the existing blobs on all the commits on all the repositories, potentially **a lot** of data. In most common cases you want to filter by commit, by reference or by repository.
+
+### tree_entries
+```sql
++-----------------+------+
+| name            | type |
++-----------------+------+
+| repository_id   | TEXT |
+| tree_entry_name | TEXT |
+| blob_hash       | TEXT |
+| tree_hash       | TEXT |
+| tree_entry_mode | TEXT |
++-----------------+------+
+```
+
+`tree_entries` table contains all the objects from all the repositories that are [tree objects](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects#_git_commit_objects).
+
+
+### files
+```sql
++-----------------+-------+
+| name            | type  |
++-----------------+-------+
+| repository_id   | TEXT  |
+| file_path       | TEXT  |
+| blob_hash       | TEXT  |
+| tree_hash       | TEXT  |
+| tree_entry_mode | TEXT  |
+| blob_content    | BLOB  |
+| blob_size       | INT64 |
++-----------------+-------+
+```
+
+`files` is an utility table mixing `tree_entries` and `blobs` to create files. It includes the file path.
```

Comment:
I would mention here that queries to this table are expensive and should be done carefully.